Hackney.Shared.CautionaryAlerts/Factories/ResponseFactory.cs

```diff
@@ -78,11 +78,18 @@ public static CautionaryAlertResponse ToResponse(this CautionaryAlertListItem do
                 Description = domain.CautionOnSystem,
                 Reason = domain.Reason,
                 AssureReference = domain.AssureReference,
-                PersonId = Guid.Parse(domain.PersonId),
+                PersonId = CheckIfExists(domain.PersonId),
                 PersonName = domain.Name
             };
         }
 
+        private static Guid? CheckIfExists(string personId)
+        {
+            if (personId == null)
+                return null;
```

Comment:
Perhaps, it would be useful to include additional conditional check of:
``` cs
if (personId == Guid.Empty)
```

What I foresee as possible is that a value within the database could be set by another piece of code - one that does not treat "personId" as a nullable value. As such, it's possible that the lack of personId within the database would be represented by the string of: "0000...-0000... etc". This code would fail to detect such case, leading to, possibly, unexpected results.